blocks/form/form.css

```diff
@@ -0,0 +1,450 @@
+:root {
+    --form-font-size-m: 22px;
+    --form-font-size-s: 18px;
+    --form-font-size-xs: 16px;
+    --form-background-color: white;
+    --form-padding: 3%;
+    --form-columns: 1;
+    --form-field-gap: 20px 0 0 0;
+    --form-invalid-border-color: #ff5f3f;
```

Comment:
please add (and other colors) to global colors list in `styles.css` and reference from there